dts/bindings/cpu/riscv,cpus.yaml

```diff
@@ -17,3 +17,7 @@ properties:
     description: RISC-V instruction set architecture
     required: true
     type: string
+
+  timebase-frequency:
+    type: int
+    description: Optional Timebase Frequency (frequency of the system timer) in Hz for this CPU
```

Comment:
Should this be directly in the `compatible = "riscv,machine-timer"; `?

and, maybe:

```diff
-    description: Optional Timebase Frequency (frequency of the system timer) in Hz for this CPU
+    description: Timebase Frequency in Hz for system timer
```

Reply:
> Yes that would also work, considered it, but the linux one is in the cpus group, what do you think is best?

We need to figure out how to add in the CPU group then.

Reply:
I had already looked at that and couldnt figure it out, went to discord and asked and was told it's likely impossible. So i have moved this property to the riscv mtimer binding, since it mostly applies to riscv cpus. BL60x mostly only serves as a exmaple while also fixing it as a issue.

Reply:
That works great, thank you.